content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -135,30 +134,25 @@ For pods, this is any pod that has a deletion timestamp set.
 
 ### Topology information {#topology}
 
-{{< feature-state for_k8s_version="v1.20" state="deprecated" >}}
+Each endpoint within an EndpointSlice can contain relevant topology information.
+The topology information includes the location of the endpoint and information
+about the corresponding Node and zone. This will be available in the following
+per endpoint fields on EndpointSlices:
+
+* `nodeName` - The name of the Node this endpoint is on.
+* `zone` - The zone this endpoint is in.
 
 {{< note >}}
-The topology field in EndpointSlices has been deprecated and will be removed in
-a future release. A new `nodeName` field will be used instead of setting
-`kubernetes.io/hostname` in topology. It was determined that other topology
-fields covering zone and region would be better represented as EndpointSlice
-labels that would apply to all endpoints within the EndpointSlice.
+In the GA API, the per endpoint `topology` was effectively removed in favor of
```

Comment:
Tiny Nit: I think "v1 API" might be slightly easier to understand, but I'm not sure if we have a standard here for docs.

Reply:
I will go with "v1 Api" for now then. I wasn't sure what the standard was here either.